src/storm/solver/GurobiLpSolver.h

```diff
@@ -17,6 +18,25 @@ int __stdcall GRBislp(GRBenv**, const char*, const char*, const char*, const cha
 namespace storm {
 namespace solver {
 
+class GurobiEnvironment {
+   public:
+    GurobiEnvironment() = default;
```

Comment:
One shouldn't be able to create a copy of this 
```suggestion
    GurobiEnvironment() = default;
    GurobiEnvironment(GurobiEnvironment const&) = delete;
    GurobiEnvironment& operator=(GurobiEnvironment const&) = delete;
```